libs/blocks/ost/ost.js

```diff
@@ -0,0 +1,28 @@
+import { loadScript, getConfig } from '../../utils/utils.js';
+
+const { miloLibs, codeRoot } = getConfig();
+const base = miloLibs || codeRoot;
+
+export default async function init() {
+  if (!window.ost) {
+    await loadScript(`${base}/deps/offer-selector-tool.js`);
+  }
+
+  const OstCountry = 'US';
```

Comment:
if the constants are not exported, shouldn't they follow camelCase?
Also can keep country and language next to each other?

ostCountry 
os**t**Language

Finally, country and language should be inferred from the context of the document, is that possible?